src/tests/Jenkinsfile

```diff
@@ -1,32 +1,59 @@
+properties([
+  pipelineTriggers([[$class: 'PeriodicFolderTrigger', interval: '1d']]),
+  parameters([
+    string(name: 'API_URL', defaultValue: 'https://api.quantum-inspire.com/', description: 'Quantum Inspire API url'),
+    string(name: 'API_CREDENTIALS_ID', defaultValue: 'api.quantum-inspire.com', description: 'Jenkins credentials ID'),
+  ])
+])
 pipeline {
     agent {
         dockerfile {
               filename 'src/tests/Dockerfile.build'
               args '-v pip_cache:/var/pip_cache'
         }
     }
+    triggers {
+        cron(env.BRANCH_NAME == "feature/DEM-722/Jenkins" ? "H H(4-7) * * 1-5" : "")
+    }
     stages {
-        stage('Build') {
+        stage('Checkout modules') {
             steps {
                 sh 'virtualenv venv'
-                sh '. ./venv/bin/activate && pip install .[qiskit,projectq] --cache-dir /var/pip_cache'
-                sh 'pwd'
-                sh 'ls -lah'
+                sh '''
+                    . ./venv/bin/activate &&
+                    pwd &&
+                    ls -lah &&
+                    # Installing qiskit, qiskit-aer and qiskit-terra as separated (development) packages doesn't work
+                    # as expected. As a workaround we install the qiskit packages from PyPI, uninstall package qiskit
+                    # and reinstall the development version of qiskit afterwards.
+                    pip install qiskit --cache-dir /var/pip_cache &&
+                    pip uninstall -y qiskit --cache-dir /var/pip_cache &&
+                    git clone https://github.com/Qiskit/qiskit &&
+                    pip install ./qiskit --upgrade --cache-dir /var/pip_cache -r ./qiskit/requirements-dev.txt
```

Comment:
This only installs the meta package, you can probably leave this out, but add `pip uninstall qiskit-terra` and then clone qiskit terra and install manually. Maybe the same for qiskit-aer and qiskit-ignis, although I think they are less important for the goal of this test (but then again, you might get incompatibility issues if you only install master of terra, and not master of the rest).

Reply:
Same on the build server with Jenkins while installing qiskit-aer: Could NOT find BLAS (missing: BLAS_LIBRARIES). I'll try without uninstalling qiskit-aer.

Reply:
Yeah you have to `apt install libblas-dev`, it's not installed by default. But not installing qiskit-aer is fine, since we don't use it for our tests anyway.

Reply:
I tried but it needs administrator rights to install which I don't have. I'll leave it out for now.